app/airq/commands/invalid.py

```diff
@@ -11,8 +11,11 @@ def should_handle(self) -> bool:
 
     def handle(self) -> typing.List[str]:
         return [
-            'Unrecognized option "{}". Reply with M for the menu{}.'.format(
-                self.user_input,
-                " or U to stop this alert" if self.client.is_enabled_for_alerts else "",
+            gettext(
+                'Unrecognized option "%(user_input)s". Reply with M for the menu%(alert_message)s.',
+                user_input=self.user_input,
+                alert_message=" or U to stop this alert"
```

Comment:
This will break right?

It's probably best to do this one as:

```python3
txt = gettext('Unrecognized option "%(user_input)s".', user_input=self.user_input)
if self.client.is_enabled_for_alerts;
    txt += gettext("Reply with M for the menu or U to stop this alert.")
else:
    txt += gettext("Reply with M for the menu.")
return [txt]